### ISSUE TITLE
[Bug]: AH01276: Cannot serve directory: No matching DirectoryIndex (v24)

### ISSUE THREAD

**Issue by Maxcodesthings** [nextcloud/server#33454]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Nextcloud Upgrade to version 24 (tried latest and stable) results in::
`AH01276: Cannot serve directory /var/www/html/apps/files/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive`

I have looked over the internet for this issue, and I seem to be the only one. I stepped on some pretty url fix but that did not work. This is docker afterall it should be easier than this to upgrade.

Full log:
```
[Wed Aug 03 22:18:31.388398 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.54 (Debian) PHP/8.0.21 configured -- resuming normal operations
[Wed Aug 03 22:18:31.388429 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
172.25.0.2 - Max [03/Aug/2022:22:18:38 -0700] "GET /status.php HTTP/1.1" 200 1033 "-" "Mozilla/5.0 (Windows) mirall/3.5.3stable-Win64 (build 20220729) (Nextcloud, windows-10.0.19044 ClientArchitecture: x86_64 OsArchitecture: x86_64)"
[Wed Aug 03 22:18:58.420879 2022] [autoindex:error] [pid 30] [client 172.25.0.2:47272] AH01276: Cannot serve directory /var/www/html/apps/files/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
172.25.0.2 - - [03/Aug/2022:22:18:58 -0700] "GET /apps/files/?dir=/&fileid=178 HTTP/1.1" 403 652 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
```

**Edit:** I notice that i went from major version 22 to 24, however the upgrade proceeded and downgrading is not possible. I get `Can't start Nextcloud because the version of the data (24.0.3.2) is higher than the docker image version (23.0.7.2) and downgrading is not supported. Are you sure you have pulled the newest image version?`

So since this is not allowed my installation gets bricked? Why does the upgrade proceed if its not allowed to upgrade multiple major versions?

Ive tried forcing the upgrade to continue in the container as well:
```
www-data@f5b31e0a2a3f:~/html$ php -d memory_limit=-1 -f /var/www/html/occ upgrade
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Setting log level to debug
Exception: Updates between multiple major versions and downgrades are unsupported.
Update failed
Maintenance mode is kept active
Resetting log level
```

**Edit2**: I have decided to try a fresh installation so i just renamed my volumes and pointed at stable image using this docker-compose:
```
 app:
    image: nextcloud:stable #Upgrading nextcloud from 22.1.1.2 ...
    container_name: nextcloud-app
    networks:
      - nextcloud_network
    depends_on:
      - acme
      - proxy
      - db
    volumes:
      - nextcloud24:/var/www/html
      - config24:/var/www/html/config
      - apps24:/var/www/html/custom_apps
      - data24:/var/www/html/data
      - themes24:/var/www/html/themes
      - /etc/localtime:/etc/localtime:ro
 ```

There is now no data in the volumes as i renamed them they are new volumes however the error i get is the same:
```
root@eschaton:/opt/nextcloud# docker logs --since 1h --follow nextcloud-app
Initializing nextcloud 24.0.3.2 ...
New nextcloud instance
Initializing finished
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.25.0.10. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.25.0.10. Set the 'ServerName' directive globally to suppress this message
[Wed Aug 03 23:14:40.313137 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.54 (Debian) PHP/8.0.21 configured -- resuming normal operations
[Wed Aug 03 23:14:40.313165 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
[Wed Aug 03 23:15:36.741729 2022] [autoindex:error] [pid 69] [client 172.25.0.2:32928] AH01276: Cannot serve directory /var/www/html/apps/files/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
172.25.0.2 - - [03/Aug/2022:23:15:36 -0700] "GET /apps/files/?dir=/&fileid=178 HTTP/1.1" 403 652 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
172.25.0.2 - Max [03/Aug/2022:23:17:32 -0700] "GET /status.php HTTP/1.1" 200 1218 "-" "Mozilla/5.0 (Windows) mirall/3.5.3stable-Win64 (build 20220729) (Nextcloud, windows-10.0.19044 ClientArchitecture: x86_64 OsArchitecture: x86_64)"
172.25.0.2 - Max [03/Aug/2022:23:17:32 -0700] "PROPFIND /remote.php/dav/files/Max/ HTTP/1.1" 302 513 "-" "Mozilla/5.0 (Windows) mirall/3.5.3stable-Win64 (build 20220729) (Nextcloud, windows-10.0.19044 ClientArchitecture: x86_64 OsArchitecture: x86_64)"
172.25.0.2 - Max [03/Aug/2022:23:17:32 -0700] "PROPFIND /index.php HTTP/1.1" 200 3239 "-" "Mozilla/5.0 (Windows) mirall/3.5.3stable-Win64 (build 20220729) (Nextcloud, windows-10.0.19044 ClientArchitecture: x86_64 OsArchitecture: x86_64)"
[Wed Aug 03 23:17:41.140087 2022] [autoindex:error] [pid 73] [client 172.25.0.2:32936] AH01276: Cannot serve directory /var/www/html/apps/files/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
```

### Steps to reproduce

1.Install older version of nextcloud, I came from version 22
2. docker stop nextcloud
3. docker rm nextcloud
4. docker pull nextcloud:stable
5. docker-compose up -d
6. Experience error in log

### Expected behavior

The error to not exist. I did not alter the .htaccess or any config file, i dont know why this would be happening.

### Installation method

Official Docker image

### Operating system

Debian/Ubuntu

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
www-data@f5b31e0a2a3f:~/html$ php -d memory_limit=-1 -f /var/www/html/occ config:list system
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
{
    "system": {
        "htaccess.RewriteBase": "\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.domain.io"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "22.1.1.2",
        "overwrite.cli.url": "http:\/\/nextcloud.domain.io",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "overwriteprotocol": "https",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpsecure": "ssl",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 2,
        "maintenance": true
    }
}
```

### List of activated Apps

```shell
www-data@f5b31e0a2a3f:~/html$ php -d memory_limit=-1 -f /var/www/html/occ app:list          
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Enabled:
  - accessibility: 1.7.0
  - activity: 2.15.0
  - apporder: 0.13.0
  - breezedark: 22.0.1
  - bruteforcesettings: 2.2.0
  - calendar: 2.3.4
  - circles: 22.1.1
  - cloud_federation_api: 1.4.0
  - comments: 1.11.0
  - contactsinteraction: 1.2.0
  - dashboard: 7.1.0
  - dav: 1.18.0
  - deck: 1.5.3
  - documentserver_community: 0.1.11
  - federatedfilesharing: 1.11.0
  - federation: 1.11.0
  - files: 1.16.0
  - files_pdfviewer: 2.3.0
  - files_rightclick: 1.1.0
  - files_sharing: 1.13.2
  - files_trashbin: 1.11.0
  - files_versions: 1.14.0
  - files_videoplayer: 1.11.0
  - firstrunwizard: 2.11.0
  - logreader: 2.7.0
  - lookup_server_connector: 1.9.0
  - mail: 1.10.5
  - news: 16.0.1
  - nextcloud_announcements: 1.11.0
  - notes: 4.1.1
  - notifications: 2.10.1
  - oauth2: 1.9.0
  - onlyoffice: 7.1.2
  - password_policy: 1.12.0
  - photos: 1.4.0
  - privacy: 1.6.0
  - provisioning_api: 1.11.0
  - recommendations: 1.1.0
  - serverinfo: 1.12.0
  - settings: 1.3.0
  - sharebymail: 1.11.0
  - spreed: 12.1.2
  - support: 1.5.0
  - survey_client: 1.10.0
  - systemtags: 1.11.0
  - text: 3.3.0
  - theming: 1.12.0
  - twofactor_backupcodes: 1.10.1
  - updatenotification: 1.11.0
  - user_status: 1.1.1
  - viewer: 1.6.0
  - weather_status: 1.1.0
  - workflowengine: 2.3.1
Disabled:
  - admin_audit
  - encryption
  - files_external
  - officeonline: 1.1.1
  - user_ldap
```

**Comment by ncabanis** [comment]
I ran into the same issue. It is fair to say only one major to the next is valid to upgrade. But then an upgrade should not be attempted, leaving the data corrupted!

**Comment by szaimen** [comment]
Hi, please update to 25.0.7 or better 26.0.2 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 26-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+

**Comment by btravouillon** [comment]
I hit the same issue today after upgrading the container from 25.0.7.1 to 26.0.3.2.

```
$ docker-compose pull
Pulling app ... done
$ docker-compose up -d
Recreating nextcloud_app_1 ... done
$ docker-compose logs -ft
Attaching to nextcloud_app_1
app_1  | 2023-07-14T21:00:33.333854080Z Initializing nextcloud 26.0.3.2 ...
app_1  | 2023-07-14T21:00:33.334008633Z Upgrading nextcloud from 25.0.7.1 ...
app_1  | 2023-07-14T21:02:18.758164532Z => Searching for scripts (*.sh) to run, located in the folder: /docker-entrypoint-hooks.d/pre-upgrade
app_1  | 2023-07-14T21:02:18.766909994Z ==> but the hook folder "pre-upgrade" is empty, so nothing to do
app_1  | 2023-07-14T21:02:19.951146238Z Nextcloud or one of the apps require upgrade - only a limited number of commands are available
app_1  | 2023-07-14T21:02:19.951255514Z You may use your browser or the occ upgrade command to do the upgrade
app_1  | 2023-07-14T21:02:19.986929739Z Setting log level to debug
app_1  | 2023-07-14T21:02:19.990959671Z Turned on maintenance mode
app_1  | 2023-07-14T21:05:16.330503255Z Updating database schema
app_1  | 2023-07-14T21:05:39.196100145Z Updated database
app_1  | 2023-07-14T21:05:39.233063718Z Updating <federation> ...
app_1  | 2023-07-14T21:05:39.279444750Z Updated <federation> to 1.16.0
app_1  | 2023-07-14T21:05:39.282111294Z Updating <lookup_server_connector> ...
app_1  | 2023-07-14T21:05:39.307589369Z Updated <lookup_server_connector> to 1.14.0
app_1  | 2023-07-14T21:05:39.310212802Z Updating <oauth2> ...
app_1  | 2023-07-14T21:05:39.340036807Z Updated <oauth2> to 1.14.0
app_1  | 2023-07-14T21:05:39.342345829Z Updating <password_policy> ...
app_1  | 2023-07-14T21:05:39.365237268Z Updated <password_policy> to 1.16.0
app_1  | 2023-07-14T21:05:39.367792496Z Updating <photos> ...
app_1  | 2023-07-14T21:05:39.403420205Z Updated <photos> to 2.2.0
app_1  | 2023-07-14T21:05:39.406401511Z Updating <files> ...
app_1  | 2023-07-14T21:05:39.484200083Z Updated <files> to 1.21.1
app_1  | 2023-07-14T21:05:39.496799461Z Updating <circles> ...
app_1  | 2023-07-14T21:05:39.690935415Z Updated <circles> to 26.0.0
app_1  | 2023-07-14T21:05:39.693652904Z Updating <cloud_federation_api> ...
app_1  | 2023-07-14T21:05:39.717007903Z Updated <cloud_federation_api> to 1.9.0
app_1  | 2023-07-14T21:05:39.719918394Z Updating <dav> ...
app_1  | 2023-07-14T21:05:39.843858755Z Fix broken values of calendar objects
app_1  | 2023-07-14T21:05:39.844022071Z 
app_1  | 2023-07-14T21:05:39.844446742Z  Starting ...
app_1  | 2023-07-14T21:05:39.847163601Z 
app_1  | 2023-07-14T21:05:39.921451066Z Clean up old calendar subscriptions from deleted users that were not cleaned-up
app_1  | 2023-07-14T21:05:39.921544381Z 
app_1  | 2023-07-14T21:05:39.921623826Z     0/0 [>---------------------------]   0% Starting ...
app_1  | 2023-07-14T21:05:39.923532863Z 
app_1  | 2023-07-14T21:05:40.023106733Z Updated <dav> to 1.25.0
app_1  | 2023-07-14T21:05:40.025434885Z Updating <files_external> ...
app_1  | 2023-07-14T21:05:40.060782015Z Updated <files_external> to 1.18.0
app_1  | 2023-07-14T21:05:40.127225143Z Updating <files_sharing> ...
app_1  | 2023-07-14T21:05:40.213438624Z Updated <files_sharing> to 1.18.0
app_1  | 2023-07-14T21:05:40.215998704Z Updating <files_trashbin> ...
app_1  | 2023-07-14T21:05:40.251741134Z Updated <files_trashbin> to 1.16.0
app_1  | 2023-07-14T21:05:40.280693241Z Updating <files_versions> ...
app_1  | 2023-07-14T21:05:45.586883120Z Updated <files_versions> to 1.19.1
app_1  | 2023-07-14T21:05:45.601577219Z Updating <sharebymail> ...
app_1  | 2023-07-14T21:05:45.627162584Z Updated <sharebymail> to 1.16.0
app_1  | 2023-07-14T21:05:45.628958360Z Updating <workflowengine> ...
app_1  | 2023-07-14T21:05:45.678131018Z Updated <workflowengine> to 2.8.0
app_1  | 2023-07-14T21:05:45.680197426Z Updating <comments> ...
app_1  | 2023-07-14T21:05:45.704272907Z Updated <comments> to 1.16.0
app_1  | 2023-07-14T21:05:45.719788976Z Updating <firstrunwizard> ...
app_1  | 2023-07-14T21:05:45.742880449Z Updated <firstrunwizard> to 2.15.0
app_1  | 2023-07-14T21:05:45.745297675Z Updating <nextcloud_announcements> ...
app_1  | 2023-07-14T21:05:45.778806243Z Updated <nextcloud_announcements> to 1.15.0
app_1  | 2023-07-14T21:05:45.780609852Z Updating <notifications> ...
app_1  | 2023-07-14T21:05:45.826912650Z Updated <notifications> to 2.14.0
app_1  | 2023-07-14T21:05:45.829209968Z Updating <theming> ...
app_1  | 2023-07-14T21:05:45.917371374Z Migrate old user accessibility config
app_1  | 2023-07-14T21:05:45.917450560Z 
app_1  | 2023-07-14T21:05:45.917666506Z     0/0 [->--------------------------]   0% Starting ...
app_1  | 2023-07-14T21:05:45.952878961Z 
app_1  | 2023-07-14T21:05:45.991591096Z Updated <theming> to 2.1.1
app_1  | 2023-07-14T21:05:45.994163472Z Updating <contactsinteraction> ...
app_1  | 2023-07-14T21:05:46.029500074Z Updated <contactsinteraction> to 1.7.0
app_1  | 2023-07-14T21:05:46.031682224Z Updating <dashboard> ...
app_1  | 2023-07-14T21:05:46.055456017Z Updated <dashboard> to 7.6.0
app_1  | 2023-07-14T21:05:46.057661796Z Updating <federatedfilesharing> ...
app_1  | 2023-07-14T21:05:46.082756212Z Updated <federatedfilesharing> to 1.16.0
app_1  | 2023-07-14T21:05:46.085004510Z Updating <files_rightclick> ...
app_1  | 2023-07-14T21:05:46.108710598Z Updated <files_rightclick> to 1.5.0
app_1  | 2023-07-14T21:05:46.110828451Z Updating <logreader> ...
app_1  | 2023-07-14T21:05:46.133622216Z Updated <logreader> to 2.11.0
app_1  | 2023-07-14T21:05:46.135622401Z Updating <privacy> ...
app_1  | 2023-07-14T21:05:46.159262248Z Updated <privacy> to 1.10.0
app_1  | 2023-07-14T21:05:46.161283878Z Updating <provisioning_api> ...
app_1  | 2023-07-14T21:05:46.184611333Z Updated <provisioning_api> to 1.16.0
app_1  | 2023-07-14T21:05:46.186539739Z Updating <recommendations> ...
app_1  | 2023-07-14T21:05:46.209266874Z Updated <recommendations> to 1.5.0
app_1  | 2023-07-14T21:05:46.211241170Z Updating <related_resources> ...
app_1  | 2023-07-14T21:05:46.233966360Z Updated <related_resources> to 1.1.0-alpha1
app_1  | 2023-07-14T21:05:46.236020435Z Updating <serverinfo> ...
app_1  | 2023-07-14T21:05:46.269406741Z Updated <serverinfo> to 1.16.0
app_1  | 2023-07-14T21:05:46.271464668Z Updating <settings> ...
app_1  | 2023-07-14T21:05:46.294785715Z Updated <settings> to 1.8.0
app_1  | 2023-07-14T21:05:46.296737233Z Updating <support> ...
app_1  | 2023-07-14T21:05:46.358213113Z Updated <support> to 1.9.0
app_1  | 2023-07-14T21:05:46.360301391Z Updating <survey_client> ...
app_1  | 2023-07-14T21:05:46.399961663Z Updated <survey_client> to 1.14.0
app_1  | 2023-07-14T21:05:46.401932811Z Updating <text> ...
app_1  | 2023-07-14T21:05:51.822953893Z Updated <text> to 3.7.2
app_1  | 2023-07-14T21:05:51.824601741Z Updating <twofactor_backupcodes> ...
app_1  | 2023-07-14T21:05:51.863531016Z Updated <twofactor_backupcodes> to 1.15.0
app_1  | 2023-07-14T21:05:51.865626146Z Updating <updatenotification> ...
app_1  | 2023-07-14T21:05:51.899714600Z Updated <updatenotification> to 1.16.0
app_1  | 2023-07-14T21:05:51.901685359Z Updating <user_status> ...
app_1  | 2023-07-14T21:05:51.938937174Z Updated <user_status> to 1.6.0
app_1  | 2023-07-14T21:05:51.940996711Z Updating <viewer> ...
app_1  | 2023-07-14T21:05:51.964232124Z Updated <viewer> to 1.10.0
app_1  | 2023-07-14T21:05:51.965910731Z Updating <weather_status> ...
app_1  | 2023-07-14T21:05:51.989512406Z Updated <weather_status> to 1.6.0
app_1  | 2023-07-14T21:05:54.548287406Z Update app calendar from App Store
app_1  | 2023-07-14T21:06:05.248431197Z Update app contacts from App Store
app_1  | 2023-07-14T21:06:17.789686075Z Starting code integrity check...
app_1  | 2023-07-14T21:06:43.338114112Z Finished code integrity check
app_1  | 2023-07-14T21:06:43.347497232Z Update successful
app_1  | 2023-07-14T21:06:43.350990107Z Turned off maintenance mode
app_1  | 2023-07-14T21:06:43.352375043Z Resetting log level
app_1  | 2023-07-14T21:06:45.655912718Z The following apps have been disabled:
app_1  | 2023-07-14T21:06:45.662231279Z => Searching for scripts (*.sh) to run, located in the folder: /docker-entrypoint-hooks.d/post-upgrade
app_1  | 2023-07-14T21:06:45.667309777Z ==> but the hook folder "post-upgrade" is empty, so nothing to do
app_1  | 2023-07-14T21:06:45.667392055Z Initializing finished
app_1  | 2023-07-14T21:06:45.667578760Z => Searching for scripts (*.sh) to run, located in the folder: /docker-entrypoint-hooks.d/before-starting
app_1  | 2023-07-14T21:06:45.673133465Z ==> but the hook folder "before-starting" is empty, so nothing to do
app_1  | 2023-07-14T21:06:45.778073565Z     0/0 [--->------------------------]   0%AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.18.0.2. Set the 'ServerName' directive globally to suppress this message
app_1  | 2023-07-14T21:06:45.825463479Z AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.18.0.2. Set the 'ServerName' directive globally to suppress this message
app_1  | 2023-07-14T21:06:45.906561881Z [Fri Jul 14 21:06:45.906159 2023] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.56 (Debian) PHP/8.2.8 configured -- resuming normal operations
app_1  | 2023-07-14T21:06:45.906664660Z [Fri Jul 14 21:06:45.906305 2023] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
app_1  | 2023-07-14T21:11:49.030134129Z [Fri Jul 14 21:11:49.029583 2023] [autoindex:error] [pid 84] [client 192.168.0.8:59178] AH01276: Cannot serve directory /var/www/html/apps/dashboard/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
app_1  | 2023-07-14T21:11:49.031213683Z 192.168.0.8 - - [14/Jul/2023:21:11:49 +0000] "GET /apps/dashboard/ HTTP/1.0" 403 664 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
app_1  | 2023-07-14T21:11:51.431855165Z 192.168.0.8 - - [14/Jul/2023:21:11:51 +0000] "GET /apps/dashboard/ HTTP/1.0" 403 664 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
app_1  | 2023-07-14T21:11:51.431995813Z [Fri Jul 14 21:11:51.430340 2023] [autoindex:error] [pid 85] [client 192.168.0.8:59180] AH01276: Cannot serve directory /var/www/html/apps/dashboard/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
```

This instance was migrated from ownCloud 10.11.0 to Nextcloud 25.0.6 a couple of months ago, then migrated to Docker with the Nextcloud Docker image 25.0.7.

**Comment by btravouillon** [comment]
Strange. Before the upgrade, I checked the version with the browser at https://nextcloud.example.tld/apps/dashboard/.

It turns out that after the upgrade, this URL does not work anymore (I was trying to refresh the page). Once I tried to access https://nextcloud.example.tld/, I've been redirected to https://nextcloud.example.tld/index.php/apps/dashboard/ which is a different URL but works perfectly well.

**Comment by joshtrichards** [comment]
Hi @Maxcodesthings - I'm sorry to see you ran into this problem. This is/was an issue with the official community NC Docker image, not with NC Server. The correct repository for this is https://github.com/nextcloud/docker not here.

From the looks of it, this was probably addressed in nextcloud/docker#2018.

Unfortunately since you're already in the situation, it can be a bit of a mess to untangle (but from the looks of it it is possible). See the issues referenced in the above PR for some approaches that have worked for other people, as well as more recent comments in nextcloud/docker#2015.

**Comment by joshtrichards** [comment]
Closing since this is a docker image issue - see/follow-up as needed in nextcloud/docker#2015.
